lib/foods.js

```diff
@@ -135,6 +147,7 @@ function createNewFood(name, calories){
 $('document').ready(function() {
   // load the data
   reloadTables();
+  updateLineListener();
```

Comment:
Actually it is only called on #food-list tds, so I think we are ok there! I am moving that call now...